share/picongpu/examples/atomicPhysics/include/picongpu/param/dimension.param

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2014-2020 Axel Huebl, Rene Widera
```

Comment:
Is this not the default? If it is, please remove this file.

Reply:
done